src/resource/notes/notes.controller.ts

```diff
@@ -24,7 +24,7 @@ export class NoteController {
         return this.notesService.fetchUserNotes(user);
     }
 
-    @Post(':id')
+    @Post()
```

Comment:
Fix is ok, but I'd leave it out of this PR. It's best that Pull Request do what they say they do and nothing more. This doesn't affect the implemented functionality.